components/tools/OmeroM/src/annotations/writeFileAnnotation.m

```diff
@@ -59,7 +63,8 @@
     originalFile.setMimetype(rstring(ip.Results.mimetype));
 end
 
-originalFile = updateOriginalFile(session, originalFile, filePath);
+originalFile = updateOriginalFile(...
+    session, originalFile, filePath, varargin{:});
```

Comment:
Copying https://github.com/snoopycrimecop/openmicroscopy/commit/3a07788f721d13ed3bbce96a31f06b7d19040c53#commitcomment-11597771

> See https://ci.openmicroscopy.org/view/Failing/job/OMERO-5.1-merge-tracking/384/console, if description (or likely namespace) is passed, it will not pass the updateOriginalFile input check